packages/runtime-core/src/hydration.ts

```diff
@@ -168,17 +169,26 @@ export function createHydrationFunctions(
           }
         } else {
           if ((node as Text).data !== vnode.children) {
-            hasMismatch = true
-            ;(__DEV__ || __FEATURE_PROD_HYDRATION_MISMATCH_DETAILS__) &&
-              warn(
-                `Hydration text mismatch in`,
-                node.parentNode,
-                `\n  - rendered on server: ${JSON.stringify(
-                  (node as Text).data,
-                )}` +
-                  `\n  - expected on client: ${JSON.stringify(vnode.children)}`,
-              )
-            ;(node as Text).data = vnode.children as string
+            let dataContent = (node as Text).data.replace(/[\t\r\n\f ]+/g, '')
```

Comment:
Only `\r` or `\r\n` would actually cause a mismatch. `\t\f` and blank will not

Reply:
done